DependencyInjection/SonataUserExtension.php

```diff
@@ -273,4 +257,55 @@ protected function aliasManagers(ContainerBuilder $container, $managerType)
         $container->setAlias('sonata.user.user_manager', sprintf('sonata.user.%s.user_manager', $managerType));
         $container->setAlias('sonata.user.group_manager', sprintf('sonata.user.%s.group_manager', $managerType));
     }
+
+    /**
+     * @param array $config
+     */
+    private function checkManagerTypeToModelTypeMapping(array $config)
+    {
+        $managerType = $config['manager_type'];
+
+        $actualModelClasses = [
+            $config['class']['user'],
+            $config['class']['group'],
+            $config['admin']['user']['class'],
+            $config['admin']['group']['class'],
+        ];
+
+        if ('orm' === $managerType) {
+            $expectedModelClasses = [
+                EntityUser::class,
+                EntityGroup::class,
+                EntityUserAdmin::class,
+                EntityGroupAdmin::class,
+            ];
+        } elseif ('mongodb' === $managerType) {
+            $expectedModelClasses = [
+                DocumentUser::class,
+                DocumentGroup::class,
+                DocumentUserAdmin::class,
+                DocumentGroupAdmin::class,
+            ];
+        } else {
+            throw new \InvalidArgumentException(sprintf('Invalid manager type "%s".', $managerType));
+        }
+
+        foreach ($actualModelClasses as $index => $actualModelClass) {
+            if ('\\' === substr($actualModelClass, 0, 1)) {
+                $actualModelClass = substr($actualModelClass, 1);
+            }
+
+            $expectedModelClass = $expectedModelClasses[$index];
+
+            if ($actualModelClass !== $expectedModelClass && !is_subclass_of($actualModelClass, $expectedModelClass)) {
```

Comment:
This line needs a break. And maybe a KitKat.

Reply:
![ba dum tss](http://media.riffsy.com/images/38069a6064edbf7d6fada1a98f96ef15/tenor.gif)

Reply:
@Soullivaneuh But its length is 119 characters. As far as I understand, this is acceptable, is not it? Maybe you mean the next line?
```php
                throw new \InvalidArgumentException(sprintf('Model class "%s" does not correspond to manager type "%s".', $actualModelClass, $managerType));
```

Reply:
Oh, indeed. Even if a line-break would be great here, I'll approve the PR so.